src/test/java/org/apache/commons/lang3/builder/DiffBuilderTest.java

```diff
@@ -32,6 +32,31 @@
     
     private static final ToStringStyle SHORT_STYLE = ToStringStyle.SHORT_PREFIX_STYLE;
 
+    private static class IdentifiableTest {
```

Comment:
Thinking about this some more, I don't think we need this class at all. We can simply use Integer instances fur der ```objectField``` of ```TypeTestClass```